bigbio/hub/hub_repos/sem_eval_2024_task_2/sem_eval_2024_task_2.py

```diff
@@ -258,6 +284,16 @@ def _generate_examples(self, data_dir: Path, split: str, config) -> Tuple[int, D
                 else:
                     premise = f" Primary: {text_primary}"
 
-                yield id_, {"id": id_, "premise": premise, "hypothesis": raw_label_data[id_]["Statement"], "label": raw_label_data[id_]["Label"]}
+                yield id_, {"id": id_, "premise": premise, "hypothesis": raw_label_data[id_]["Statement"], "label": raw_label_data[id_].get("Label")}
         else:
             raise ValueError(f"Unknown schema {self.config.schema}")
+
+    def _load_split_file(self, data_dir_or_path, split):
```

Comment:
Add parameter and return type datatype hints

Reply:
Should be fixed with 450a5494e6fe2a968bb5dfede34193c095c41784